tests/yam/agama/boot_agama.pm

```diff
@@ -47,6 +47,9 @@ sub prepare_boot_params {
         set_var('INST_AUTO', $profile_url);
         push @params, "inst.auto=\"$profile_url\"", "inst.finish=stop";
     }
+    if (get_var('HDD_2') =~ /oemdrv/) {
```

Comment:
maybe new setting `INST_OEMDRV=1` would be more clear. And do it in a single line, it is more readable.